cloud-init: add manpage

powersj opened the pull request.

paride commented:
+1 for the manpages. Should we add the `REPORTING BUGS` and `SEE ALSO` sections pointing to the Launchpad bugs page and to the readthedocs documentation? Check the very end of the `ls` manpge to see what I have in mind.

I'd also mention the cloud-init licenses in the COPYRIGHT section, something like:

>  Copyright 2020 Canonical Ltd. Released under the GNU GPL version 3 or under the Apache License version 2.0. On Ubuntu and Debian systems the full text of the licenses is available in </usr/share/common-licenses/GPL-3> and </usr/share/common-licenses/Apache-2.0>.

powersj commented:
I have added the SEE ALSO section and added license info to the COPYRIGHT section. However, I am -1 on the REPORTING BUGS section as that various depending on if it is upstream or package bug. I thought about pointing to the bugs docs, but would rather keep the man page simple.
